crates/forge_main/src/completer/input_completer.rs

```diff
@@ -49,7 +49,7 @@ impl Completer for InputCompleter {
                                 style: None,
                                 extra: None,
                                 span: query.span,
-                                append_whitespace: false,
+                                append_whitespace: true,
```

Comment:
why was this changed?

Reply:
Its a bit irritating to add whitespace after embedding a file.